dsgrid/registry/dimension_mapping_registry_manager.py

```diff
@@ -162,9 +163,16 @@ def validate_records(self, config: DimensionMappingsConfig):
                 mapping.mapping_type.value,
             )
 
-            if mapping.archetype.check_fraction_sum_eq1:
+            if mapping.archetype.check_fraction_sum_eq1_from_id:
```

Comment:
Does this change affect any mappings in the project repositories?

Reply:
no, this is only a name change. We've only checked sum group by from_id for all mappings so far.